### ISSUE TITLE
regression bug: deb package can't be launched anymore

### ISSUE THREAD

**Issue by wiiznokes** [crabnebula-dev/cargo-packager#119]
I think #117 broke deb somehow.

When i launch my app, i get this error message:
```
$ fan-control
bash: /usr/bin/fan-control: Permission non accordée
```
(permission denied)

There is no auto completion. 

```
$ sudo /usr/bin/fan-control
sudo: /usr/bin/fan-control : commande introuvable

$ ls /usr/bin/fan-control
/usr/bin/fan-control
```
(command not found)

I tested to revert the commit, and it worked again

@naman-crabnebula

**Comment by naman-crabnebula** [comment]
Hi @wiiznokes ,

Thanks a lot for reporting. It should be fixed with #120 . Can you please test it and report, if possible.

Thank You,
With Regards
Naman Garg

**Comment by wiiznokes** [comment]
Hi, i just tested it and it's work again, thanks you
